templates/paragraphs/paragraph--oe-text-feature-media.html.twig

```diff
@@ -14,15 +14,19 @@
     '#alt':        image.alt,
   } %}
 {% endif %}
+{% if content.field_oe_text_long['#items'] %}
```

Comment:
No because this settings deppends of the full text field is fulfilled or not, so by default is false but if the full text field has some content, the settings must be true.